client/rest/src/plugins/rosetta/accountRoutes.js

```diff
@@ -28,21 +29,49 @@ import BlockIdentifier from './openApi/model/BlockIdentifier.js';
 import Coin from './openApi/model/Coin.js';
 import CoinIdentifier from './openApi/model/CoinIdentifier.js';
 import {
-	RosettaErrorFactory, createLookupCurrencyFunction, rosettaPostRouteWithNetwork
+	RosettaErrorFactory, createLookupCurrencyFunction, rosettaPostRouteWithNetwork, stitchBlockTransactions
 } from './rosettaUtils.js';
 import { NetworkLocator } from 'symbol-sdk';
 import { Network } from 'symbol-sdk/symbol';
 
 export default {
 	register: (server, db, services) => {
+		const PAGE_SIZE = 100;
+
 		const networkName = services.config.network.name;
 		const network = NetworkLocator.findByName(Network.NETWORKS, networkName);
 		const lookupCurrency = createLookupCurrencyFunction(services.proxy);
 		const getChainHeight = () => services.proxy.fetch('chain/info', json => json.height);
 		const getBlockIdentifier = height => services.proxy.fetch(`blocks/${height}`)
 			.then(blockInfo => new BlockIdentifier(Number(blockInfo.block.height), blockInfo.meta.hash));
+		const parser = new OperationParser(network, {
+			includeFeeOperation: true,
+			lookupCurrency,
+			resolveAddress: (unresolvedAddress, transactionLocation) =>
```

Comment:
for consistency would rename unresolvedAddress => address or change other places where OperationParser is created. could even add function to rosettaUtils for creating OperationParser for data routes